src/DynamoCoreWpf/ViewModels/PackageManager/PackageManagerSearchViewModel.cs

```diff
@@ -634,8 +639,6 @@ private void PopulateMyPackages()
             // We should have already populated the CachedPackageList by this step
             if (PackageManagerClientViewModel.CachedPackageList == null ||
                 !PackageManagerClientViewModel.CachedPackageList.Any()) return;
-            // We need the user to be logged in, otherwise there is no point in runnig this routine
-            if (PackageManagerClientViewModel.LoginState != Greg.AuthProviders.LoginState.LoggedIn) return;
```

Comment:
Shouldn't we return here in PopulateMyPackages() if the user is not logged in?

Reply:
We are, we are! This was an odd one, I don't know if it was me who coded this previously, but we had this check twice in a roll, which doesn't make any sense. Check out a tiny bit further down line `464`: 

https://github.com/DynamoDS/Dynamo/pull/14610/files/bb856d807375c41efd392535fda5206197e5915a#diff-bb521c95d6777ba5ae6d31b92bdd9cf63d44eae8b06941efac80648554ecc287R646

The only difference is that we are initializing the `SearchMyResults`, so it's not a `null` anymore, which allows us to set the screen later on.